Source/Core/DolphinQt/MainWindow.ui

```diff
@@ -159,14 +165,6 @@
     <string>Stop</string>
    </property>
   </action>
-  <action name="actionAboutQt">
-   <property name="text">
-    <string>About &amp;Qt</string>
-   </property>
-   <property name="menuRole">
-    <enum>QAction::AboutQtRole</enum>
```

Comment:
I don't really see any reason why we can't leave the About Qt dialog in; it's not causing any problems, and they make an awesome toolkit, so who cares if it's somewhat advert-ish?